.github/workflows/ci-pkg-install.yml

```diff
@@ -17,141 +17,40 @@ defaults:
 
 jobs:
 
-  init-temp:
-    runs-on: ubuntu-20.04
-    steps:
-    - run: mkdir pypi && touch pypi/.placeholder
-    - uses: actions/upload-artifact@v3
-      with:
-        name: ci-packages-${{ github.sha }}
-        path: pypi
-
-  install-standalone:
-    needs: init-temp
+  install-pkg:
     runs-on: ${{ matrix.os }}
     strategy:
-      fail-fast: true
-      max-parallel: 1
+      fail-fast: false
       matrix:
         os: [ubuntu-20.04, ubuntu-22.04, macOS-11, macOS-12, windows-2022]
-        pkg: ["app", "lite", "pytorch"]
-        python-version: [3.8]  # , 3.9
+        pkg: ["app", "lite", "pytorch", ""]
```

Comment:
Can this use "lightning" as everywhere else?

Reply:
I thought they were the same. What is the difference?

Reply:
They have the effect, but from practical point of view, almost no user will have set this env variable by default to "lightning"

Reply:
@Borda Let's use lightning everywhere. The CI is already confusing enough for new users.